Resources/Prototypes/Entities/Mobs/Species/human.yml

```
@@ -26,107 +26,1224 @@
     - Hair
     - Snout
   - type: Inventory
+### Female
```

YAMLlint (1.37.1)

[warning] comment not indented like content

(comments-indentation)